src/blocks/modal.css

```diff
@@ -48,6 +48,10 @@
   max-width: fit-content;
 }
 
```

Comment:
* The `confirmation` and `avatar` modals contain incorrect `font`. Please, add `font-family: "Poppins", Arial, sans-serif;` for all modals in `modal.css` in line 2


```css

.modal {
  font-family: "Poppins", Arial, sans-serif;
  position: fixed;
  top: 0;
// and so on
```